src/clients/TokenClient.ts

```diff
@@ -185,7 +185,7 @@ class TokenClient {
 
     const symbol = this.getSymbol(denom);
     if (TokenClient.isPoolTokenLegacy(denom)) {
-      const match = symbol.match(/^([a-z\d.-\/]+)-(\d+)-([a-z\d.-\/]+)-(\d+)-lp\d+$/i);
+      const match = symbol.match(/^([a-z\d.-/]+)-(\d+)-([a-z\d.-/]+)-(\d+)-lp\d+$/i);
```

Comment:
is this regex ok? seems like there's an error
![image](https://github.com/Switcheo/carbon-js-sdk/assets/7552714/ecf1a8cb-5f25-482c-a879-f346e8d7eb9a)